### ISSUE TITLE
Configure production server for new Praise setup

### ISSUE THREAD

**Issue by kristoferlund** [givepraise/praise#896]
- API
- Discord Bot
- Frontend
- DB

**Comment by mhmdksh** [comment]
> Do we run all services on one server to begin with?
> 
> * API
> * Discord Bot
> * Frontend
> * DB
>   
>   * Replication
> 
> DB replication should be on separate server, right @mhmdksh ?

Yes exactly it should be on another entity (Either Server, or a Managed Cloud DB env)

**Comment by kristoferlund** [comment]
@mhmdksh you can go ahead and start setting up the production server. Let's keep the setup as simple as possible to begin with. That means, if we can support 100 communities on a local mongodb then no need to configure a db cluster right now.

First launch then upgrade.

**Comment by mhmdksh** [comment]
This has been done, I'm closing this issue. Check: https://github.com/givepraise/saas/issues/3
